src/interfaces/correction-filing-interface.ts

```diff
@@ -21,7 +23,7 @@ export interface CorrectionFilingIF {
   correction: {
     correctedFilingId: string
     correctedFilingType: string
-    correctedFilingDate: string
+    correctedFilingDate: FormattedDateTimeGmt
```

Comment:
FormattedDateTimeGmt is an alias to a string, but the name gives us extra meaning.

In shared repo:
```
/**
  * Formatted date-time string with explicit GMT timezone (ie, UTC).
  * @example "Tue, 19 Apr 2022 16:40:18 GMT"
  */
export type FormattedDateTimeGmt = string
```